test/images/images.test.js

```diff
@@ -0,0 +1,102 @@
+import { expect } from 'chai';
+import Joi from 'joi';
+import { imageUpload } from '../../src/images';
+
+describe('Image Schemas', () => {
+  describe('Mutable Fields', () => {
+    it('should pass validation given a correct image object', () => {
+      let correctImage =
+        {
+          description: 'test',
+          categories: ['test1', 'test2'],
+          isDefault: true,
+          order: 0
+        };
+
+      let result = Joi.validate(correctImage, imageUpload.mutableFields('Mutable Fields'));
+      expect(result.error).to.be.null;
```

Comment:
A tip for the future, you can use [joi.attempt](https://github.com/hapijs/joi/blob/v8.0.5/API.md#attemptvalue-schema-message) to remove some of these `expect(result.error).to.be.or.not.to.be` because joi.attempt will throw an exception if it fails. Another way you could do it:
```
expect(() =>{
  joi.attempt(obj, schema);
}).to.throw(/my expected message/);
```

This allows for matching the message that is thrown to help make sure that you aren't just blowing up on anything.